### PR TITLE
Ch setup gh ci

### PR DESCRIPTION
**Pull request by Patricknshimiyimana**
**What does this PR Do?**
>  Continuous Integration & pipeline

**Description of Task to be completed**
> setup testing environment with jest & enzyme
> setup workflow for Continuous Integration
> Add code climate & Github action badges
> configured heroku pipeline on heroku

**How should this be manually tested?**
> - git clone to `https://github.com/atlp-rwanda/elite-bn-frontend.git`
> - checkout to `ch-setup-gh-ci`
> - `npm install` to install node_modules
> - `npm start ` to start server

**What are the relevant Trello story?**
> https://trello.com/c/IZJUnGJ2/26-setup-continuous-integration-quality-assurance-pipeline

**Screenshot**
> Screen Shot 2021-01-26 at 11 55 04

Screen Shot 2021-01-26 at 11 56 54

Screen Shot 2021-01-26 at 11 58 49

### CODE REVIEW

File: enzyme.config.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/** Used in jest.config.js */
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```
Comment (nicele08) [inline enzyme.config.js:5]: add new line

**Review by nicele08 — APPROVED** [review]

**Comment by b0nbon1** [comment]
Please always squash commits before you push

**Comment by b0nbon1** [comment]
Please could you rebase and update your branch? you have conflicts

**Review by 444root — APPROVED** [review]

**Review by NIRoberto — APPROVED** [review]
After rebase and resolve conflict will be LGTM

**Review by nicele08 — APPROVED** [review]

**Review by pextech — APPROVED** [review]

**Pull request merged**